collector/socket/src/main/cfg/org.apache.karaf.decanter.collector.socket.cfg

```diff
@@ -8,3 +8,8 @@
 
 # Protocol tcp(default) or udp
 #protocol=tcp
+
+# Dataformat json(default) : 
+# - The dataformat has to match the unmarshaller property in the component annotation
+# - Example : property = Marshaller.SERVICE_KEY_DATAFORMAT + "=json"
```

Comment:
The comment is not correct as it's the target. I would set:

```
# Unmarshaller to use
# Unmarshaller is identified by data format. The default is json, but you can use another unmarshaller
# unmarshaller.target=(dataFormat=json)
```
Also not, it's better to use `dataFormat` instead of `dataformat`.